tests/system/testdata/authz/default/deployments.txtar

```diff
@@ -0,0 +1,67 @@
+user zumi
+
+ak project create --name pz
+return code == 0
+capture_jq zpid .project_id
+
+ak project create --name pc --org cats
+return code == 0
+capture_jq cpid .project_id
+
+ak project deploy $zpid --file main.star 
+return code == 0
+capture_jq zdid .[].deployment_id | select (.)
+
+user gizmo
+
+ak project deploy $zpid --file main.star 
+return code == $RC_UNAUTHZ
+
+ak project deploy $cpid --file main.star 
+return code == 0
+capture_jq cdid .[].deployment_id | select (.)
+
+ak deployment deactivate $zdid
+return code == $RC_UNAUTHZ
+
+ak deployment deactivate $cdid
+return code == 0
+
+user zumi
+ak deployment deactivate $cdid
+return code == 0
+
+ak deployment list --project $zpid
+return code == 0
+
+ak deployment list --project $cpid
+return code == 0
+
+user gizmo
+
+ak deployment list --project $zpid
+return code == $RC_UNAUTHZ
```

Comment:
Shouldn't list for someone else's stuff also return NOT FOUND instead of UNAUTHZ?